book_maker/cli.py

```diff
@@ -308,6 +313,7 @@ def main():
         is_test=options.test,
         test_num=options.test_num,
         prompt_config=parse_prompt_arg(options.prompt_arg),
+        no_bilingual=options.no_bilingual,
```

Comment:
if we add here we also need to add it in txt loader

Reply:
I want people who use this loader to modify the loader they use.(I only use epub now) How about writing the currently supported loader in this help?